tests/store/artifact/test_azure_blob_artifact_repo.py

```diff
@@ -3,8 +3,11 @@
 import pytest
 from unittest import mock
 
+try:
+    from azure.storage.blob import BlobPrefix, BlobProperties
```

Comment:
Assuming we're leveraging the latest version of `azure-storage-blob`, I don't think it's 100% necessary to try/catch here.

Reply:
Good point, will take out